packages/codemods/nuxt/4/default-data-error-value/LICENSE

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2024 mohab-sameh
```

Comment:
@35C4n0r if it's going to be committed to our repo then the license should be to Codemod Inc.